.github/workflows/python-test.yml

```diff
@@ -17,8 +17,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.7', '3.8', '3.9' ]
-        
+        python-version: [ '3.9' ]
```

Comment:
What prompted this change?

Reply:
Pipenv only supports a single Python version, so I decided to move testing to the same version that's supported by pipenv.

I can move back testing to 3.7+ but long term I'd like to switch to v3.9 completely.